### PR TITLE
Order book: Touch up design / modal size.

### PR DESCRIPTION
**Pull request by biocom**
Screenshot 2020-04-09 at 10 57 20

Desktop only for now. Mobile in progress.

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr856--dexreact.review.gnosisdev.com

File: src/styles/global.ts [hunk 1]
```diff
@@ -1088,6 +1089,30 @@ const GlobalStyles = createGlobalStyle`
...
   }
+  
+  // Heavier specification of the selector to ensure override from global.ts
+  // Ideally take out the modali default injected styles and only load from our end.
+  body.modali-open .modali.modali-size-large {
+    width: 80vw;
+    max-width: initial;
+    min-width: initial;
+    max-height: 100%;
+      @media (min-width: 500px) {
```
Comment (W3stside) [inline src/styles/global.ts:1100]: can we make this bad boy an `em`

Comment (anxolin) [reply src/styles/global.ts:1100]: I'll do it in my branch

Comment (biocom) [reply src/styles/global.ts:1100]: Actually, I did an exact override as using 50rem didn't seem to override the native 500px styles. Correct me if I'm wrong though.

**Pull request merged**